concise-mid-tag.cddl

```diff
@@ -136,8 +136,16 @@ tagged-ueid-type = #6.48000(ueid-type)
 impl-id-type = bytes .size 32
 tagged-impl-id-type = #6.47115(impl-id-type)
 
+$measured-element-type-choice /= null
+$measured-element-type-choice /= tagged-oid-type
+$measured-element-type-choice /= tagged-uuid-type
+
+measurement-map = [
+  $measured-element-type-choice, measurement-values-map
```

Comment:
Is the first value in the record '$measured-element-type-choice' still optional? Should '+' be '*'?

Reply:
OBE